src/gui/guiFormSpecMenu.cpp

```diff
@@ -4821,12 +4821,18 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 	}
 
 	if (event.EventType == EET_GUI_EVENT) {
-		if (event.GUIEvent.EventType == gui::EGET_TAB_CHANGED
-				&& isVisible()) {
-			// find the element that was clicked
+		const s32 caller_id = event.GUIEvent.Caller->getID();
+		bool close_on_enter;
```

Comment:
This appears to only be used in one case, so probably add braces to that case and move the variable there.

Reply:
I actually moved the variable outside such that I do not need to add another indent level just for that.